### ISSUE TITLE
The `Log in with Google` button is not shown on omnivore.app/login

### ISSUE THREAD

**Issue by searene** [plateaukao/einkbro#255]
**What device and app version are you using**
 - Device: Boox Tab8
 - OS: Android 11
 - EinkBro Version: 10.6.3

**Describe the bug**
The `Log in with Google` button is not shown on https://omnivore.app/login

**To Reproduce**
1. Open https://omnivore.app/login in einkbro
2. Notice that the `Log in with Google` button is missing.

This is the screenshot taken from my eink device, in which a button is missing.
![Screenshot_20230521-101243](https://github.com/plateaukao/einkbro/assets/3351317/49407d0d-4fb2-4218-885a-08e30bb403d6)

This is the screenshot taken from my macOS' chrome browser (emulated as pixel 5 with the help of the `Developer Tool`)

<img width="180" alt="Screenshot 2023-05-21 at 10 17 17" src="https://github.com/plateaukao/einkbro/assets/3351317/a963c78b-cd4c-441c-bd62-bfcc276942ca">

Notice that there's a button called `Continue as Joey`, which is missing in my eink device's screenshot.

**Additional context**
1. I've also tried the system browser on my eink device, no problem. The button is shown correctly. It seems that the issue can only be reproduced in einkbro.
2. Both `White background` and `Black font` are disabled.
3. AdBlock is disabled.

**Comment by plateaukao** [comment]
It works on my Tab UltraC, Nova Air C and other non-Eink devices. 
Could you re-launch the app and try again, or see if you can login Google account in www.google.com?

**Comment by searene** [comment]
Hi, @plateaukao

Thanks for your reply! I tried restarting the browser, clearing the data, reinstalling the browser, and restarting the device, but none of them worked. In the end, I debugged for a while on my computer and found out that the browser would request https://accounts.google.com/gsi/client before showing the button. So, I manually requested this URL in einkbro and after that, the button showed itself. I'm not sure why it would solve the problem but I'm glad it's solved.

Still, thank you for taking the time to test the issue on your devices, I use einkbro every day, it's the best e-ink browser I've ever used. Thank you for creating such an amazing app!
